SOP/SOP-push.adoc

```diff
@@ -32,9 +32,7 @@ All alerts should appears as green in the Prometheus Alert Monitoring.
 
 == Prometheus Alerts Procedures
 
-The following guide outlines the steps required to solve issues which will be alert by the Prometheus alerts configured to monitoring the  https://github.com/aerogear/aerogear-unifiedpush-server[UnifiedPush Server] when it is managed, installed and configured via the https://github.com/aerogear/unifiedpush-operator[UnifiedPush Operator].
-
-IMPORTANT: Before check any following steps see if the operator pod is running successfully. More info: https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+IMPORTANT: Before checking any of following steps see if the operator pod is running successfully. More info: https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
```

Comment:
Should/could this be a file link to that other document, since they live side-by-side? That could remove the need to reference the (outdated) version in the URL.

Reply:
The link is important for the SRE checked it. It is important to remeber that the SOPs should be made thinking in someone that never saw it before. So, besides, be side-side the person will not guess it n POV. Also, note that we have already planned to create a var for all <tags> so it will not be a problem at all. For now, a find/replace from IDE should help us to do it easily.

Reply:
I'm not suggesting that we remove the link, I'm just suggesting that it could be a local one rather than a remote one.

Reply:
ah ok ˆ

Reply:
I've made the change as suggest. I've verified myself that it works but @grdryn could you verify it quickly and then I will merge this one. :)

Reply:
Verified! :100: